Training/Training/file.c

```diff
@@ -7,7 +7,51 @@
 // ------------------------------------------------------------------------------------------------
 
 #include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdint.h>
+#include <ctype.h>
+
+#define MAX 50
+
+void ChangeRequired (int* balance, int* change1, int* change2, int* change3, int* change4) {
+   int num = *balance;
+   while (num >= 10) {
+      *change1 = num / 10;
+      num = num % 10;
+   }
+   while (num >= 5) {
+      *change2 = num / 5;
+      num = num % 5;
+   }
+   while (num >= 2) {
+      *change3 = num / 2;
+      num = num % 2;
+   }
+   while (num >= 1) {
+      *change4 = num / 1;
+      num = num % 1;
+   }
+}
 
 void main () {
-	printf ("Hello World");
-}
\ No newline at end of file
+   char actAmount[MAX] = "";
+   char cashPaid[MAX] = "";
+   char* endptr1 = NULL;
+   char* endptr2 = NULL;
+   printf ("Enter the actual amount: ");
+   char* inpPtr1 = fgets (actAmount, sizeof (actAmount), stdin);
+   printf ("Enter the cash paid: ");
+   char* inpPtr2 = fgets (cashPaid, sizeof (cashPaid), stdin);
+   if (inpPtr1 == NULL || inpPtr2 == NULL) {
```

Comment:
You should also check for negative amounts, and also the case where the amount given is LESS than the bill amount.